models/_generate_layers.py

```diff
@@ -253,6 +254,9 @@ def module_call_needs_suppress_shadow_builtin_warning(self):
     LinearLayer, ConvLayer, TransposedConvLayer, RecLayer, RnnCellLayer,
     PositionalEncodingLayer, RelativePositionalEncodingLayer}
 
+  _LayerClassesWithExplicitTarget={
```

Comment:
See failing tests.
`returnn_common/models/_generate_layers.py:257: WEAK WARNING Pep8CodeStyle: E225: missing whitespace around operator`

```suggestion
  _LayerClassesWithExplicitTarget = {
```